scss/layout/_works.scss

```diff
@@ -1,3 +1,15 @@
+/**
+* works.scss
+*
+* This is the Work Experience Layout were we want to proof
+* the user and potential employes of our experience, relaiability
+* and accountability.
+*
+*
+*/
+
```

Comment:
- [ ] You're having a lot of extra lines in your code files. Kindly detect them and delete them all.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections from each other.
This makes your code cleaner and optimized.

Reply:
Ok! Thanks for your suggestion.